src/firebase_functions/private/_identity_fn.py

```diff
@@ -64,11 +64,14 @@ def _auth_user_info_from_token_data(token_data: dict[str, _typing.Any]):
 
 def _auth_user_metadata_from_token_data(token_data: dict[str, _typing.Any]):
     from firebase_functions.identity_fn import AuthUserMetadata
+    creation_time = _dt.datetime.utcfromtimestamp(int(token_data["creation_time"]) / 1000.0)
+    last_sign_in_time = None
+    if "last_sign_in_time" in token_data:
+        last_sign_in_time = _dt.datetime.utcfromtimestamp(int(token_data["last_sign_in_time"]) / 1000.0)
```

Comment:
```suggestion
        last_sign_in_time = _dt.datetime.utcfromtimestamp(int(token_data["last_sign_in_time"] / 1000.0 ))
```
Think it should be this.